ledger/appcow_test.go

```diff
@@ -413,7 +413,7 @@ func TestCowBuildDelta(t *testing.T) {
 	a.Equal(
 		basics.EvalDelta{
 			GlobalDelta: basics.StateDelta{},
-			LocalDeltas: map[uint64]basics.StateDelta{0: {}},
+			LocalDeltas: map[uint64]basics.StateDelta{},
```

Comment:
Is there another test case needed for empty sender delta + non-empty other account deltas?

Reply:
Not really needed, but added this test to document this scenario